spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/saml2/Saml2RelyingPartyProperties.java

```diff
@@ -50,13 +50,26 @@
 	 */
 	public static class Registration {
 
+		/**
+		 * Relying Party identifier URI pattern.
+		 */
+		private String localEntityIdTemplate;
```

Comment:
I think this property could be initialised with the value `{baseUrl}/saml2/service-provider-metadata/{registrationId}` as that's the default in RelyingPartyRegistration.Builder`